engines/python/setup/djl_python/tests/test_rolling_batch.py

```diff
@@ -237,13 +235,10 @@ def get_tokenizer(self):
 
         req = Request(132,
                       "This is a wonderful day",
-                      parameters={
-                          "max_new_tokens": 256,
-                          "details": True
-                      },
+                      parameters={"max_new_tokens": 256},
+                      details=True,
```

Comment:
Yes it would break the UX. So details won't be part of parameters anymore?

Reply:
this doesn't break the CX for the user. We do parse details from parameters, but we're just passing it in separately now to the Request rather than reading it from parameters within the request